Data_Structures_Questions.md

```diff
@@ -2,19 +2,19 @@ Answer the following questions for each of the data structures you implemented a
 
 ## Queue
 
-1. What is the runtime complexity of `enqueue`?
+1. What is the runtime complexity of `enqueue`? O(n)
```

Comment:
If the add to tail method that is used within the enqueue is constant, then how is the runtime complexity of the enqueue method be O(n)?